src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -183,7 +183,9 @@ export class MongoCollectionTreeItem extends AzureParentTreeItem<IMongoTreeRoot>
 		if (!document) {
 			throw new Error("The insert command requires at least one argument");
 		}
-		const insertResult = await this.collection.insert(document);
+
+		// insert is deprecated, so use insertOne instead
+		const insertResult = await this.collection.insertOne(document);
```

Comment:
Was this intentional? Seems unrelated

Reply:
It was just something I noticed when trying to resolve this issue (since I thought the bug might have stemmed from using an unsupported insert command).  I can remove it though

Reply:
I'd prefer if you removed it just so I don't have to investigate that file haha. At the moment I'm not exactly sure what's going on there